pkg/config/pipeline.go

```diff
@@ -153,25 +153,21 @@ func (p *PipelineConfig) Update(request *livekit.StartEgressRequest) error {
 		VideoBitrate: 4500,
 	}
 
-	p.updateUploadConfig()
+	p.updateUploadConfig(request)
+	connectionInfoRequired := true
 
 	switch req := request.Request.(type) {
 	case *livekit.StartEgressRequest_RoomComposite:
 		p.Info.Request = &livekit.EgressInfo_RoomComposite{RoomComposite: req.RoomComposite}
 		p.Info.RoomName = req.RoomComposite.RoomName
-		if p.Info.RoomName == "" {
```

Comment:
Is this check already made somewhere else or is it not needed?

Reply:
moved further down